sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1390,6 +1552,19 @@ std::vector<node> get_root_nodes() const;
 ----
 |Returns a list of all nodes in the graph which have no dependencies.
 
+|
+[source,c++]
+----
+size_t get_mem_alloc_size() const;
```

Comment:
I think we should consider renaming this to something like `get_required_mem_size`, to avoid it being interpreted as the amount which is currently allocated by the memory pool.